include/RevProc.h

```diff
@@ -156,6 +155,9 @@ class RevProc{
   ///< RevProc: Get pointer to Load / Store queue used to track memory operations
   std::shared_ptr<std::unordered_map<uint64_t, MemReq>> GetLSQueue(){ return LSQueue; }
 
+  // Find a Hart to assign ThreadID to
+  void AssignThread(const uint32_t& ThreadID);
```

Comment:
A reference is not required for a trivial type like `uint32_t`, and causes extra memory operations.